### ISSUE TITLE
Meta Types break

### ISSUE THREAD

**Issue by flyingsilverfin** [graknlabs/client-java#110]
## Description
Asking `tx.getMetaEntityType()` breaks

## Environment
Grakn Core 1.8 (master) and client-java 1.8 (master, 6074059cbaf6eb5cb4e564ef950be68a1653b9c2).

## Reproducible Steps
```

    @Test
    public void testMeta() {
        GraknClient.Session testing = client.session("testing");
        GraknClient.Transaction tx = testing.transaction().write();
        tx.execute(Graql.parse("define person sub entity;").asDefine());

        MetaType.Remote<?, ?> metaEntityType = tx.getMetaEntityType();
    }
```

## Expected Output
Get the meta "entity" type.

## Actual Output
Exception: 
```
The concept [grakn.client.concept.type.impl.EntityTypeImpl.Remote{tx=grakn.client.GraknClient$Transaction@5ce8d869, id=V4152}] is not of type [interface grakn.client.concept.type.MetaType$Remote]
grakn.client.concept.GraknConceptException: 
	at grakn.client.concept.GraknConceptException.create(GraknConceptException.java:35)
	at grakn.client.concept.GraknConceptException.invalidCasting(GraknConceptException.java:42)
	at grakn.client.concept.Concept$Remote.asMetaType(Concept.java:559)
	at grakn.client.GraknClient$Transaction.getMetaEntityType(GraknClient.java:770)
	at grakn.client.test.integration.answer.AnswerIT.testMeta(AnswerIT.java:74)
```
 
## Additional information
I thought we had tests that should hit this case?

**Comment by adammitchelldev** [comment]
I was sure we did

**Comment by adammitchelldev** [comment]
Fixed by #111
